crates/router/src/services/api.rs

```diff
@@ -145,6 +145,32 @@ pub trait ConnectorIntegration<T, Req, Resp>: ConnectorIntegrationAny<T, Req, Re
         Ok(ErrorResponse::get_not_implemented())
     }
 
+    fn get_server_error_response(
+        &self,
+        res: types::Response,
+    ) -> CustomResult<ErrorResponse, errors::ConnectorError> {
+        let error_mssg = match res.status_code {
+            500 => "InternalServerError",
+            501 => "NotImplemented",
+            502 => "BadGateway",
+            503 => "ServiceUnavailable",
+            504 => "GatewayTimeout",
+            505 => "HttpVersionNotSupported",
+            506 => "VariantAlsoNegotiates",
+            507 => "InsufficientStorage",
+            508 => "LoopDetected",
+            510 => "NotExtended",
+            511 => "NetworkAuthenticationRequired",
+            _ => "UnknownError",
```

Comment:
Can this be converted to lowercase? Since in all our apis we have lowercase / snake_case values.